api/libproject/devstart.rb

```diff
@@ -220,6 +250,7 @@ def start_local_api()
   setup_local_environment
   common = Common.new
   common.status "Starting API server..."
+  ENV["OVERWRITE_WORKBENCH_DB_HOST"] = "true"
```

Comment:
What does this line do? I can't find any references to this string elsewhere.

Reply:
ah, should have been removed. good catch